Portfolio/index.html

```diff
@@ -38,7 +38,7 @@
     <div class="i">

```

Comment:
The opening line does not seem to belong to the code. It appears to be a description of the developer's skills. Please include only the actual code in the provided code block.